src/ButtonsGroup.tsx

```diff
@@ -19,7 +19,7 @@ export namespace ButtonsGroupProps {
         alignment?: "left" | "center" | "right";
         /** Default: false */
         buttonsEquisized?: boolean;
-        buttons: [ButtonProps, ButtonProps, ...ButtonProps[]];
```

Comment:
Sometimes in DSFR, button groups can have only one button, like for confirm button in consent modal.